### PR TITLE
feat: Add Homosaurus

### LINKED ISSUE #502: Add Homosaurus
https://homosaurus.org

### PR DESCRIPTION
**Pull request by ddeboer**
* Merge all PoolParty SKOS queries.
* Expose `skos:related` for CHT, too.

Fix #502

### CODE REVIEW

File: packages/network-of-terms-catalog/catalog/datasets/abr.jsonld [hunk 1]
```diff
@@ -41,7 +41,7 @@
         },
         {
           "@type": "FindAction",
-          "query": "file://catalog/queries/lookup/abr-cht.rq"
+          "query": "file://catalog/queries/lookup/poolparty.rq"
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/datasets/abr.jsonld:44]: Alternatively, the more cumbersome `abr-cht-homosaurus`, but there may be something to say against entering tech vendors in our filenames.

File: packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "https://homosaurus.org/",
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld:3]: Or should this be https://data.ihlia.nl/homosaurus?

File: packages/network-of-terms-catalog/catalog/queries/lookup/poolparty.rq [hunk 3]
```diff
@@ -45,5 +47,10 @@ WHERE {
         ?narrower_uri skos:prefLabel ?narrower_prefLabel .
         FILTER(LANG(?narrower_prefLabel) = "nl")
     }
+    OPTIONAL {
+        ?uri skos:related ?related_uri .
+        ?related_uri skos:prefLabel ?related_prefLabel .
+        FILTER(LANG(?related_prefLabel) = "nl")
+    }
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/queries/lookup/poolparty.rq:54]: Related is also relevant for searching CHT, which I’ll add in a separate PR.

File: packages/network-of-terms-catalog/catalog/queries/search/poolparty.rq [hunk 4]
```diff
@@ -6,14 +6,14 @@ CONSTRUCT {
         skos:altLabel ?altLabel ;
         skos:hiddenLabel ?hiddenLabel ;
         skos:scopeNote ?scopeNote ;
-        skos:exactMatch ?skos_exactMatch ;
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/queries/search/poolparty.rq:9]: Wasn’t being used, so removed.

File: packages/network-of-terms-catalog/catalog/queries/search/poolparty.rq [hunk 5]
```diff
@@ -45,7 +45,9 @@ WHERE {
         FILTER(LANG(?narrower_prefLabel) = "nl")
     }
     OPTIONAL {
-        ?uri skos:exactMatch ?skos_exactMatch .
+        ?uri skos:related ?related_uri .
+        ?related_uri skos:prefLabel ?related_prefLabel .
+        FILTER(LANG(?related_prefLabel) = "nl")
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/queries/search/poolparty.rq:50]: Relevant for CHT too, which didn’t have `skos:related` yet.

File: packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld (hunk 2 above)
Comment (sdevalk) [reply packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld:3]: I think it should be https://data.ihlia.nl/homosaurus - but I'll verify!

File: packages/network-of-terms-catalog/catalog/queries/search/poolparty.rq (hunk 5 above)
Comment (sdevalk) [reply packages/network-of-terms-catalog/catalog/queries/search/poolparty.rq:50]: 👍🏻

File: packages/network-of-terms-catalog/catalog/datasets/abr.jsonld (hunk 1 above)
Comment (sdevalk) [reply packages/network-of-terms-catalog/catalog/datasets/abr.jsonld:44]: Good one. I think `poolparty.rq` is fine: if we would continue with the names of the datasets, that could result in unwieldy filenames (`abr-cht-homosaurus-anothersource-yetanothersource-andyetanothersource.rq`)

Comment (sdevalk) [reply packages/network-of-terms-catalog/catalog/datasets/abr.jsonld:44]: On second thought: `poolparty.rq` could suggest that this file contains one query for all sources in PoolParty. But that isn't so: e.g. the WO2 Thesaurus has its own file.

File: packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+      "@id": "https://ihlia.nl"
+    }
+  ],
+  "url": [
+    "https://data.ihlia.nl/homosaurus/"
+  ],
+  "description": [
+    {
+      "@language": "nl",
+      "@value": "LGBTQ+ erfgoed"
```
Comment (sdevalk) [inline packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld:22]: I'll check this with IHLIA.

Comment (sdevalk) [reply packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld:22]: The preferred description: 

> Termen voor het beschrijven van LHBTIQ (Lesbisch/Homo/Biseksueel/Transgender/Intersekse/Queer)-publicaties en -erfgoed

**Comment by sdevalk** [comment]
> There’s still a slight problem with the URIs: we have https://homosaurus.org/v3/homoit0000025 but also https://data.ihlia.nl/homosaurus/homoit0000025.html being used as URI. Should we consider the former the old (and no longer relevant) setup?

Answer: yes, `https://homosaurus.org/...` is the old one. The leading version is `https://data.ihlia.nl/homosaurus/...`.

File: packages/network-of-terms-catalog/catalog/datasets/abr.jsonld (hunk 1 above)
Comment (ddeboer) [reply packages/network-of-terms-catalog/catalog/datasets/abr.jsonld:44]: True, but assuming query editors start looking in the `dataset-name.jsonld`, they will find the right query file there.

**Pull request merged**